src/redux/selectors.js

```diff
@@ -14,18 +14,20 @@ const selectActiveTokenAccount = (state) => {
     loginType: state.wallet.loginType,
   };
 };
-const selectAccountBalance = state => ( // @todo account has multiple balance now
-  state.wallet.info ? state.wallet.info[state.token.active].summary.balance : undefined);
-const selectBookmarks = state => state.bookmarks[state.token.active];
+const selectAccountBalance = (
+  state // @todo account has multiple balance now
+) => (state.wallet.info ? state.wallet.info[state.token.active].summary.balance : undefined);
+const selectBookmarks = (state) => state.bookmarks[state.token.active];
 const selectBookmark = (state, address) =>
-  state.bookmarks[state.token.active].find(item => (item.address === address));
-const selectSettings = state => state.settings;
-const selectNetwork = state => state.network;
-const selectServiceUrl = state => state.network.networks?.LSK?.serviceUrl;
-const selectNetworkIdentifier = state => state.network.networks?.LSK?.networkIdentifier;
-const selectNetworkName = state => state.network.name;
-const selectActiveTokenNetwork = state => state.network.networks[state.token.active];
-const selectStaking = state => state.staking;
+  state.bookmarks[state.token.active].find((item) => item.address === address);
+const selectSettings = (state) => state.settings;
+const selectNetwork = (state) => state.network;
+const selectServiceUrl = (state) => state.network.networks?.LSK?.serviceUrl;
+const selectNetworkIdentifier = (state) => state.network.networks?.LSK?.networkIdentifier;
+const selectNetworkName = (state) => state.network.name;
+const selectActiveTokenNetwork = (state) => state.network.networks[state.token.active];
+const selectStaking = (state) => state.staking;
+const selectHWAccounts = (state) => state.settings.hardwareAccounts;
```

Comment:
HW = hardaware wallet
```suggestion
const selectHWAccounts = (state) => state.settings.HWAccounts;
```

Reply:
@soroushm Hardware wallet is part of settings. You even proposed some code with it `const {hardwareAccounts} = store.settings` in https://github.com/LiskHQ/discussions/discussions/99

Reply:
the hw name is part of the setting witch we dont want to change it
the accounts it should be store in the hw domain under "/src/modules/hardwareWallet/store"

Reply:
Fixed